### PR TITLE
Add servers list documentation

### LINKED ISSUE #58: Servers list documentation
GET method documentation from /api/servers endpoint.

### PR DESCRIPTION
**Pull request by danielrcoura**
This issue fix #58.

### CODE REVIEW

File: routes/servers.js [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+   *
+   * @apiDescription
+   * Lists all servers that the client can connect to have access to user directories.
+   *
+   * @apiSuccess {String} servers An array with server names, addresses, and ports.
+   *
+   * @apiSuccessExample {json} Success
+   * HTTP/1.1 200 OK
+   * {
+   *    "servers": [
```
Comment (dfquaresma) [inline routes/servers.js]: Could you please add lcc3 too?

File: docs/api_project.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   "apidoc": "0.3.0",
   "generator": {
     "name": "apidoc",
-    "time": "2018-09-19T12:39:27.244Z",
+    "time": "2018-09-20T00:23:35.215Z",
```
Comment (dfquaresma) [inline docs/api_project.json]: Why must we keep time information?

File: docs/api_project.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ define({
   "apidoc": "0.3.0",
   "generator": {
     "name": "apidoc",
-    "time": "2018-09-19T12:39:27.244Z",
+    "time": "2018-09-20T00:23:35.215Z",
```
Comment (dfquaresma) [inline docs/api_project.js]: Why must we keep time information?

File: docs/api_data.js [hunk 4]
```diff
@@ -297,5 +297,36 @@ define({ "api": [
...
+          {
+            "group": "Success 200",
+            "type": "String",
+            "optional": false,
+            "field": "servers",
+            "description": "<p>An array with server names, addresses, and ports.</p>"
+          }
+        ]
+      },
+      "examples": [
```
Comment (dfquaresma) [inline docs/api_data.js]: Could you please add lcc3 too?

File: docs/api_data.json [hunk 5]
```diff
@@ -297,5 +297,36 @@
...
+            "type": "String",
+            "optional": false,
+            "field": "servers",
+            "description": "<p>An array with server names, addresses, and ports.</p>"
+          }
+        ]
+      },
+      "examples": [
+        {
+          "title": "Success",
```
Comment (dfquaresma) [inline docs/api_data.json]: Could you please add lcc3 too?

File: docs/api_project.json (hunk 2 above)
Comment (danielrcoura) [reply docs/api_project.json]: The docs folder is generated automatically by [apiDocs](http://apidocjs.com/).

File: docs/api_data.js (hunk 4 above)
Comment (danielrcoura) [reply docs/api_data.js]: Done!

File: routes/servers.js (hunk 1 above)
Comment (danielrcoura) [reply routes/servers.js]: Done!

File: docs/api_data.json (hunk 5 above)
Comment (danielrcoura) [reply docs/api_data.json]: The docs folder is generated automatically by [apiDocs](http://apidocjs.com/).

File: docs/api_project.js (hunk 3 above)
Comment (danielrcoura) [reply docs/api_project.js]: The docs folder is generated automatically by [apiDocs](http://apidocjs.com/).

**Pull request merged**